src/common/transformations/src/transformations/fp16_compression/align_mixed_fp32_fp16_types.cpp

```diff
@@ -48,6 +49,7 @@ bool ov::pass::AlignMixedFP32FP16Types::run_on_model(const std::shared_ptr<ov::M
                 copy_runtime_info(incoming_node, convert);
                 input.replace_source_output(convert);
                 disable_fp16_compression(convert);
+                mark_as_compression(convert);
```

Comment:
This converts are decompression converts: they upcast to fp32 for precision sensitive subgraphs. Is it possible to rename `mark_as_compression` to avoid confusions? 
Since `mark_as_compression` is used only for converts that are inserted to align types for f16 and f32 parts, can we name it e.g. `mark_type_aligning_convert` to avoid confusion?